docs/operator/getting-started.md

```diff
@@ -0,0 +1,274 @@
+# Getting Started
+
+An official Helm chart is planned to make it really easy to deploy the Grafana Agent
+Operator on Kubernetes. For now, things must be done a little manually.
+
+## Deploying CustomResourceDefinitions
+
+The current set of CustomResourceDefinitions can be found in
+[production/operator/crds](../../production/operator/crds). Apply them from the
+root of this repository using:
+
+```
+kubectl apply -f production/operator/crds
+```
+
+This step *must* be done before installing the Operator, as the Operator will
+fail to start if the CRDs do not exist.
+
+## Installing on Kubernetes
+
+Use the following Deployment to run the Operator, changing values as desired:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana-agent-operator
+  namespace: default
+  labels:
+    app: grafana-agent-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grafana-agent-operator
+  template:
+    metadata:
+      labels:
+        app: grafana-agent-operator
+    spec:
+      serviceAccountName: grafana-agent-operator
+      containers:
+      - name: operator
+        image: grafana/agent-operator:v0.15.0
```

Comment:
(This image doesn't exist yet, if you're going to test this out locally you'll need to `make agent-operator-image` , re-tag it, and apply it to the k3d cluster using `k3d image import -c agent-operator grafana/agent-operator:v0.15.0`)